src/optimizer/OptimizerUtils.cpp

```diff
@@ -338,7 +334,7 @@ Value OptimizerUtils::boundValueWithMax(const meta::cpp2::ColumnDef& col, const
             return Value(std::numeric_limits<double>::max());
         }
         case Value::Type::BOOL: {
-            return v;
+            return Value(true);
```

Comment:
I think , better move the BOOL to below of switch block. let's unify that with boundValueWithLT and boundValueWithGT . WDYT?

Reply:
ok, make sense